src/actions/slack/slack.ts

```diff
@@ -124,7 +125,7 @@ https://github.com/looker/actions/blob/master/src/actions/slack/README.md`,
     const paginatedChannels = await pageLoaded([], await slack.channels.list(options))
     const channels = paginatedChannels.filter((c: any) => c.is_member && !c.is_archived)
     const reformatted: Channel[] = channels.map((channel: any) => ({id: channel.id, label: `#${channel.name}`}))
-    return reformatted
+    return _.sortBy(reformatted, "label")
```

Comment:
we can probably pull this into the form in the definition of the fields and do 1 sort.

also, can we just use js vs. including underscore?

`channels.sort((a, b) => (a.label.localeCompare(b.label)))`

Reply:
Ah nice, this is much better. This would make DMs appear before public channels but maybe something like this could work

```
channels.sort((a, b) =>  (a.label.startsWith("#") && b.label.startsWith("@") ? -1 : a.label.localeCompare(b.label)))
```

Reply:
It ends up checking for order with > sorts # and @ the desired way so I just used that